app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -201,6 +201,36 @@ class Settings private constructor(
         true
     )
 
+    fun setUseStrictTrackingProtection() {
+        preferences.edit()
+            .putBoolean(
+                appContext.getPreferenceKey(R.string.pref_key_tracking_protection_standard),
+                false
+            )
+            .apply()
+        preferences.edit()
```

Comment:
Nit: iirc, we can make both of these changes in the same `apply`

Reply:
duh 🤦‍♀️